loader/loader.go

```diff
@@ -145,10 +145,7 @@ func (w *Worker) run(ctx context.Context, fileJobQueue chan *fileJob, workerWg *
 					}
 				})
 
-				failpoint.Inject("LoadDataSlowDown", func(val failpoint.Value) {
-					ms, _ := val.(int)
-					time.Sleep(time.Duration(ms) * time.Millisecond)
-				})
+				failpoint.Inject("LoadDataSlowDown", func() {})
```

Comment:
It's better to user nil closure because of empty function will cause line number inconsistent. e.g.

```go
failpoint.Inject("LoadDataSlowDown", func() {})
// GENERATE
if _, ok := failpoint.Eval(_curpkg_("LoadDataSlowDown")); ok {
}

// TO KEEP LINE NUMBER CONSISTENT
failpoint.Inject("LoadDataSlowDown", nil)
// GENERATE
failpoint.Eval(_curpkg_("LoadDataSlowDown"))
```